bindings/matrix-sdk-ffi/src/encryption.rs

```diff
@@ -326,4 +351,20 @@ impl Encryption {
 
         Ok(result?)
     }
+
```

Comment:
Uniffi nowadays forwards doc comments over the FFI, are you sure you don't want any docs here?

Reply:
Oh I didn't realise that's a thing now 👍